client/src/main/java/com/alibaba/nacos/client/config/impl/ClientWorker.java

```diff
@@ -482,12 +484,20 @@ public void run() {
 
     private void init(Properties properties) {
 
-        timeout = Math.max(NumberUtils.toInt(properties.getProperty(PropertyKeyConst.CONFIG_LONG_POLL_TIMEOUT),
+        this.timeout = Math.max(NumberUtils.toInt(properties.getProperty(PropertyKeyConst.CONFIG_LONG_POLL_TIMEOUT),
```

Comment:
I think NumberUtils can be replace by ConvertUtils

Reply:
Okay, I will replace it.